interfaces/cython/cantera/test/test_composite.py

```diff
@@ -417,6 +454,33 @@ def check(a, b):
         b.restore_data(data)
         check(a, b)
 
+    def test_import_no_norm_water(self):
+        outfile = self.test_work_path / "solutionarray.h5"
+        # In Python >= 3.8, this can be replaced by the missing_ok argument
+        if outfile.is_file():
+            outfile.unlink()
+
+        w = ct.Water()
+        w.TQ = 300, 0.5
+        states = ct.SolutionArray(w, 5)
+        states.write_hdf(outfile)
```

Comment:
With the change to the default value for the `normalize` argument, this no longer tests the case where `normalize=False`, right?